### PR TITLE
Feature - Telemetry

### PR DESCRIPTION
**Pull request by Camwyn**
requires https://github.com/the-events-calendar/tribe-common/pull/1907

### CODE REVIEW

File: src/Tickets/Site_Health/Info_Section.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+	/**
+	 * Adds our default section to the Site Health Info tab.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $info The debug information to be added to the core information page.
+	 *
+	 * @return array The debug information to be added to the core information page.
+	 */
+	public function add_fields() {
```
Comment (redscar) [inline src/Tickets/Site_Health/Info_Section.php:83]: Is this the correct docblock? I see that there is a`@param` for `$info` but that isn't being sent into the method.

File: src/Tickets/Telemetry/Provider.php [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+	/**
+	 * Ensures the admin control reflects the actual opt-in status.
+	 * Note this filter is defined twice with different signatures.
+	 * We take the "low road" - 2 params and test them in the later function
+	 * to ensure we're only changing the thing we expect.
+	 *
+	 * @since TBD
+	 *
+	 * @param mixed  $value  The value of the attribute.
+	 * @param string $field  The field object id.
```
Comment (redscar) [inline src/Tickets/Telemetry/Provider.php:103]: Should `$field` be `$id` instead?

File: src/Tickets/Telemetry/Provider.php [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+	}
+
+	/**
+	 * Filters the exit questionnaire shown during plugin deactivation/uninstall.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string,mixed> $args The default args.
+	 *
+	 * @return array<string,mixed> $args The modified args.
```
Comment (redscar) [inline src/Tickets/Telemetry/Provider.php:131]: Should the second `$args` be `$slug` instead?

File: src/Tickets/Telemetry/Telemetry.php [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+	}
+
+	public function filter_exit_interview_args( $args, $slug ) {
+		if ( self::$plugin_slug !== $slug ) {
+			return $args;
+		}
+
+		$new_args = [
+			'plugin_logo'     => tribe_resource_url( 'images/tec-tickets-logo-text.svg', false, null, Tribe__Tickets__Main::instance() ),
+			'plugin_logo_alt' => 'TEC Common Logo',
```
Comment (zbtirrell) [inline src/Tickets/Telemetry/Telemetry.php:175]: avoid using "TEC" abbreviation with customers

Comment (zbtirrell) [reply src/Tickets/Telemetry/Telemetry.php:175]: also, this is the Event Tickets logo

**Pull request closed without merging**